src/main/java/io/vertx/core/eventbus/impl/EventBusImpl.java

```diff
@@ -407,7 +406,7 @@ protected void checkStarted() {
   }
 
   protected String generateReplyAddress() {
-    return Long.toString(replySequence.incrementAndGet());
+    return EventbusAddresses.generateInternal("reply");
```

Comment:
keep the sequence, but just prefix with "__vertx.reply."

Reply:
Ah, is the sequence used for something?

Reply:
I don't know, but I think it's less expensive than creating an UUID :-)